### PR TITLE
Update service fabric runtime (10.1.2175.9590) and sdk (7.1.2175)

### PR DESCRIPTION
**Pull request by williamoconnorme**
# Description

This change updates the service fabric runtime and service fabric SDK versions

## Classification

> Put an `x` in the box that applies.

| Selection | Change Type | Description |
| :---: | :--- | :--- |
| [ ] | Bug Fix | Non-breaking change which fixes an bug |
| [x] | Feature Request | Non-breaking change which new behaviour |
| [ ] | Breaking Change | Fix or feature that would cause existing behaviour not to work as expected |

## Checklist

> Put an `x` in the boxes that apply.

| Selection | Description |
| :---: | :--- |
| [x] | I have read the contributing document. |
| [ ] | Style and unit tests pass locally with my changes. |
| [ ] | I have added tests that prove my fix is effective or that my feature works. |
| [x] | I have added necessary documentation (if appropriate). |
| [ ] | Any dependent changes have been merged and published in downstream modules. |

## Further Details

> If this is a relatively large or complex change, please explain why this solution was chosen and what alternatives were considered.

### CODE REVIEW

**Comment by williamoconnorme** [comment]
Hi @darinegan, would it be possible to get this pushed to chocolatey after review

File: automatic/service-fabric-sdk/service-fabric-sdk.nuspec [hunk 1]
```diff
@@ -18,13 +18,13 @@ Service Fabric is a distributed systems platform that makes it easy to package,
     <licenseUrl>https://github.com/Microsoft/service-fabric/blob/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <iconUrl>https://cdn.jsdelivr.net/gh/darinegan/chocolatey-packages@latest/icons/service-fabric.png</iconUrl>
-    <releaseNotes>[Microsoft Azure Service Fabric 10.0 Release Notes](https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_10.md)</releaseNotes>
+    <releaseNotes>[Microsoft Azure Service Fabric Release Notes](https://github.com/microsoft/service-fabric/tree/master/release_notes)</releaseNotes>
```
Comment (darinegan) [inline automatic/service-fabric-sdk/service-fabric-sdk.nuspec]: Let's update the URL to the specific version release notes.

File: automatic/service-fabric/service-fabric.nuspec [hunk 2]
```diff
@@ -18,7 +18,7 @@ Service Fabric is a distributed systems platform that makes it easy to package,
     <licenseUrl>https://github.com/Microsoft/service-fabric/blob/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <iconUrl>https://cdn.jsdelivr.net/gh/darinegan/chocolatey-packages@latest/icons/service-fabric.png</iconUrl>
-    <releaseNotes>[Microsoft Azure Service Fabric 10.0 Release Notes](https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_10.md)</releaseNotes>
+    <releaseNotes>[Microsoft Azure Service Fabric Release Notes](https://github.com/microsoft/service-fabric/tree/master/release_notes)</releaseNotes>
```
Comment (darinegan) [inline automatic/service-fabric/service-fabric.nuspec]: Same as above; I believe this is the specific release notes for this version.
https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_101CU3.md

**Review by darinegan — CHANGES_REQUESTED** [review]

File: automatic/service-fabric/service-fabric.nuspec (hunk 2 above)
Comment (williamoconnorme) [reply automatic/service-fabric/service-fabric.nuspec]: Thanks, I updated to link to MD file. I had initially pointed to release notes tree as the cadence of release notes on github isn't always updated around the same time as the Microsoft docs site with download links. Same with the releases page page https://learn.microsoft.com/en-us/azure/service-fabric/release-notes#service-fabric-101-releases

File: automatic/service-fabric-sdk/service-fabric-sdk.nuspec (hunk 1 above)
Comment (williamoconnorme) [reply automatic/service-fabric-sdk/service-fabric-sdk.nuspec]: This is updated now

**Comment by darinegan** [comment]
Thanks William. When running the script `.\update_all.ps1` locally from the root of the repo, it errs out trying to parse release notes to generate the package. Could you take a look?

**Comment by williamoconnorme** [comment]
@darinegan It appears the validator in the package tools has not been working for several releases due to Microsoft changing the formatting of release notes in the markdown files. Would need to get some time to look at this but at a glance it seems the runtime and sdk pattern is now invalid while the release notes markdown tables are little bit more complex with cells containing multiple rows for each platform https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_101CU3.md

**Review by darinegan — APPROVED** [review]

**Comment by darinegan** [comment]
@williamoconnorme thanks for the heads-up; let's take this regression separately. I approved and merged this PR then published the packages to Chocolatey; they are pending review then should be available thereafter. Thanks again for your help on this.

**Pull request merged**